course/templates/course/course_base.html

```diff
@@ -75,6 +75,45 @@ <h4>{% trans "Calendar" %}</h4>
                 {% endif %}
             </div>
         {% endif %}
+        {% if query_language %}
+            {% language user_language %}
+            <div lang={{user_language}}>
+                <div class="alert alert-default">
+                    <img class="language-icon" src="/static/images/uls.svg" alt="">
+                    {% trans "You are browsing this course in a different language than is set in your preferences." %}
+                    {% if user_course_data %}
+                        <div class="button-container">
+                            <form action="{{ url_without_language }}">
+                                <button type="submit" class="aplus-button--secondary aplus-button--md">
+                                    {% blocktrans trimmed with lang=user_language|language_name_translated %}
```

Comment:
Should all of these forms include the `{% csrf_token %}`? Why is it included in only one?

Reply:
Oh right, some forms use HTTP GET while some POST.

Reply:
That is the only one of those with `post` method, which needs the token